src/test/java/nextstep/courses/domain/SessionTest.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+import nextstep.users.domain.NsUserTest;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class SessionTest {
+
+    @DisplayName("유료강의는 최대수강인원을 초과할 수 없다")
+    @Test
+    void validateUserLimitForPaidCourse() {
+        List<NsUser> users = List.of(NsUserTest.JAVAJIGI, NsUserTest.SANJIGI);
+
+        Assertions.assertThatThrownBy(() ->
+                        new Session(users, UsageType.PAY, SessionStatus.RECRUITING, 1)
+                                .validateUserLimitForPaidCourse())
```

Comment:
예외 검증시에는 테스트하고자 하는 대상만 람다식으로 표현해 주세요
```suggestion
        Session session = new Session(users, UsageType.PAY, SessionStatus.RECRUITING, 1);
        
        Assertions.assertThatThrownBy(() -> session.validateUserLimitForPaidCourse())
```